app/_hub/SmartParkingTechnology/google-logging/index.md

```diff
@@ -0,0 +1,135 @@
+---
+name: Kong Google Cloud Logging
+publisher: SmartParkingTechnology
+
+categories:
+  - logging
+
+type: plugin
+
+desc: Logs Kong requests with Google Cloud Logging
+
+description: |
+  Plugin for exporting Kong request data such as Service name, Route name, Consumer name or request latency to Google Cloud Logging.
+
+support_url: https://github.com/SmartParkingTechnology/kong-google-logging-plugin/issues
+
+source_url: https://github.com/SmartParkingTechnology/kong-google-logging-plugin
+
+license_type: Apache-2.0
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.2.x
+      - 2.1.x
+
+params:
+  name: google-logging
+  api_id: true
```

Comment:
```suggestion
  api_id: false
```
Just one remaining concern from me: the `api` entity has been deprecated since CE 0.13.0 and EE 0.32.

Reply:
Thanks I copied it from a different plugin and didn't really knew what its for...